### PR TITLE
Adding support for oshu manor grave worms

### PR DESCRIPTION
**Pull request by jayRyan24**
Also fixing some logic and adding support for more game feedback.

### CODE REVIEW

File: oshu_manor.lic [hunk 1]
```diff
@@ -28,29 +38,25 @@ class Oshu
...
-  def through?
-    return unless Flags['grave_exit']
-    Flags.reset('grave_exit')
-    fix_standing
-    puzzle_begin
-  end
-
   def grave_digger
     @graves.each do |grave|
       walk_to(grave) if Room.current.id != grave
```
Comment (rpherbig) [inline oshu_manor.lic]: The `if` is not necessary here.

File: oshu_manor.lic [hunk 2]
```diff
@@ -10,6 +10,17 @@ class Oshu
   include DRCI
 
   def initialize
+    arg_definitions = [
+      [
+        { name: 'critter', options: %w(worms kartais), description: 'What critter you hunting?' }
+      ],
+      []
```
Comment (rpherbig) [inline oshu_manor.lic]: What should this be doing if no args are provided?

File: oshu_manor.lic [hunk 3]
```diff
@@ -28,29 +38,25 @@ class Oshu
...
+    case bput('dig grave', 'You struggle to dig', 'You use your bare hands')
+    when 'You struggle to dig'
+      waitrt?
+      grave_digger
+    when 'You use your bare hands'
+      waitrt? 
+      waitfor('You dig to the surface')
+      pause 3
+      fix_standing
+      return if @critter['worms']
```
Comment (rpherbig) [inline oshu_manor.lic]: I think this can just be `@critter` (and line 57)

File: oshu_manor.lic [hunk 4]
```diff
@@ -10,6 +10,17 @@ class Oshu
   include DRCI
 
   def initialize
+    arg_definitions = [
+      [
+        { name: 'critter', options: %w(worms kartais), description: 'What critter you hunting?' }
```
Comment (rpherbig) [inline oshu_manor.lic]: Another option would be:
```
arg_definitions = [
  [
    { name: 'worm', regex: /^worm/i, description: 'Hunt grave worms' }
  ],
  [
    { name: 'kartais', regex: /^kartais/i, description: 'Hunt seordhevor kartais' }
  ]
]
```

Do you think the output from that is any more user-friendly?

**Comment by rpherbig** [comment]
Have you tested entering and exiting both kertais and worms?

**Comment by jayRyan24** [comment]
Yeah, was the last thing I did before commiting last night.

**Pull request merged**